src/Expr.js

```diff
@@ -143,6 +143,13 @@ var exprToString = function(expr, caller) {
   var keys = Object.keys(expr)
   var fn = keys[0]
 
+  // Versioned queries/lambdas will have an api_version field.
+  // We want to prevent it from being parsed and displayed as:
+  // Query(ApiVersion("3", "X", Var("X")))
+  if (fn === 'api_version') {
```

Comment:
there are no guaranties that `api_version` field will be the first attribute

Reply:
@marrony Makes sense. I'll update the PR and ping you when it's ready for review. Thanks for all your help 🙏

Reply:
**NOTE:** We are going to enact these changes in a separate PR 👍